src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -100,6 +100,13 @@ func Setup(
 			return writers.SendEvent(req, device, vars["roomID"], vars["eventType"], vars["txnID"], &emptyString, cfg, queryAPI, producer)
 		}),
 	)
+	r0mux.Handle("/rooms/{roomID}/state/{eventType}/",
```

Comment:
Picking up from the discussion in the gorilla repo this could probably also be `"/rooms/{roomID}/state/{eventType:[a-z]+(?:\\/)?}"` (making assumptions on what `eventType` looks like, but I guess you should be able to adjust that).